src/Kernel32.Tests.Shared/UseCultureAttribute.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+using System;
+using System.Globalization;
+using System.Reflection;
+using System.Threading;
+using Xunit.Sdk;
+
+/// <summary>
+/// Apply this attribute to your test method to replace the
+/// Thread.CurrentThread <see cref="CultureInfo.CurrentCulture" /> and
+/// <see cref="CultureInfo.CurrentUICulture" /> with another culture.
+/// </summary>
+[AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = false, Inherited = true)]
+internal class UseCultureAttribute : BeforeAfterTestAttribute
```

Comment:
This whole attribute seems to no-op on profile111. This should at least be documented. But it feels like we shouldn't even define the attribute in Profile111 if it's going to be useless. 
And going back to my question about why tests are using this attribute *and* passing in 0x409 to the methods they call, I don't understand why this attribute exists, if it can't provide consistent behavior or behavior we even want to test.